data/core.yaml

```diff
@@ -418,6 +417,7 @@ en:
     choose: Select feature type
     results: "{n} results for {search}"
     reference: View on OpenStreetMap Wiki
+    edit_reference: Edit or translate on OSM Wiki
```

Comment:
Is this string used anywhere?

Reply:
Good catch!  It got lost in a merge accident :)